redbot/cogs/mod/mod.py

```diff
@@ -322,7 +322,7 @@ def __unload(self):
         except discord.errors.Forbidden:
             await ctx.send(_("I'm not allowed to do that."))
         except Exception as e:
-            print(e)
+            print(str(e).encode('utf-8', errors='ignore'))
```

Comment:
I don't think this does what you think it does. You're going to be printing a `bytes` object. There must be a better way to solve #1366

Reply:
Forgot to attatch decodes as well, but the alternative solution is just to set environment vars in the launcher (`PYTHONIOENCODING='UTF8`) either that or ignore it because this is fixed in python3.6 https://www.python.org/dev/peps/pep-0528/

Reply:
Latest commit removes the `.encode` and uses the encoding param in `str()` (rather than encode + decode)